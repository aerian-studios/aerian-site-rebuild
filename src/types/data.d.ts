import * as React from "react";

export const isImageSharp = (
    image: ImageSharp | string
): image is ImageSharp => {
    return typeof image !== "string";
};

export interface Image {
    alt?: string;
    image: string | ImageSharp;
}

export interface ImageSharp {
    childImageSharp: {
        sizes: ImageSharpSizes;
        src: string;
    };
}

export interface ImageSharpSizes {
    aspectRatio?: number;
    sizes?: string;
    src: string;
    srcSet?: string;
    tracedSVG?: string;
}
