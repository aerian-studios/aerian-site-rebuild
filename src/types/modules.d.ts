declare module "*.svg";
declare module "*.png";
declare module "*.json";

// For storyshots
declare module "babel-plugin-require-context-hook/register";

// for maps
declare module "pigeon-overlay";
declare module "pigeon-maps";
