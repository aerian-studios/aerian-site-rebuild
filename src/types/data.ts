import { Stats } from "fs";
import { ParsedPath } from "path";

export type ImageField = string | ImageSharp;
export type FileField = string | Partial<FileNode>;

// Images

export interface GalleryImage {
    alt?: string;
    image: ImageField;
}

export interface FileNode extends ParsedPath, Stats {
    publicURL: string;
    id: string;
    children: any[];
    sourceInstanceName: string;
    absolutePath: string;
    relativePath: string;
    relativeDirectory: string;
    extension: string;
    size: number;
    prettySize: string;

    modifiedTime: string;
    accessTime: string;
    changeTime: string;
    birthTime: string;
    internal: {
        contentDigest: string;
        type: "Directory" | "File";
        mediaType?: string;
        description: string;
    };
}

export interface ImageSharp extends Partial<FileNode> {
    childImageSharp: {
        fluid?: ImageSharpFluid;
        fixed?: ImageSharpFixed;
    };
}
export interface ImageSharpFluid {
    aspectRatio?: number;
    sizes?: string;
    src: string;
    srcSet?: string;
    tracedSVG?: string;
    base64?: string;
}

export interface ImageSharpFixed {
    aspectRatio?: number;
    src: string;
    srcSet?: string;
    tracedSVG?: string;
    base64?: string;
    width: number;
    height: number;
}

// PAGES

export interface NodeList<T> {
    edges: Array<Edge<T>>;
}

export interface Edge<T> {
    node: T;
}
export interface PagesListData {
    data: {
        allPagesJson: PageList;
    };
}

export interface PageList {
    edges: PageListNode[];
}

export type PageListNode = Edge<PageRef>;

export interface PageRef {
    id: string;
    path: string;
    title: string;
}

export interface PageSection {
    title: string;
    image?: ImageField;
    smallImage?: ImageField;
    subtitle: string;
    blurb: string;
    activities?: Activity[];
    successStory?: SuccessStory;
    testimonial?: Testimonial;
}

export interface Page {
    title: string;
    path: string;
    heroImage?: ImageField;
    subheading: string;
    description: string;
    seoTitle: string;
    seoDescription: string;
    seoKeywords: string;
}

export interface WhatWeDo extends Page {
    sections?: PageSection[];
}

export interface MeetTheTeam extends Page {
    staff: Staff[];
}

export interface About extends Page {
    infographic?: Infographic[];
}

export type InfographicType =
    | "Single Line"
    | "Compact Single Line"
    | "Split"
    | "Bar"
    | "Vertical"
    | "Horizontal"
    | "Surround";
export interface Infographic {
    primaryText?: string;
    type: InfographicType;
    secondaryText?: string;
    image?: ImageField;
    imageCount: number;
    secondaryImage?: ImageField;
}

export interface Staff {
    name: string;
    jobTitle: string;
    live: boolean;
    imageNormal: ImageField;
    imageFunny: ImageField;
    description: string;
    fact: string;
    skills: string[];
}

export interface ReactRouterLocation {
    action: string;
    hash: string;
    key: string;
    pathname: string;
    search: string;
    state: null | string;
}

export interface Testimonial {
    quote?: string;
    person?: string;
    title?: string;
    avatar?: ImageField;
    image?: ImageField;
}

export interface Stat {
    title: string;
    text: string;
    image: ImageField;
}
export interface SuccessStory {
    title: string;
    stats?: Stat[];
}

export interface Activity {
    title: string;
    text: string;
}

export interface Hurdle {
    title: string;
    image: ImageField;
    text: string;
}

export interface ProjectStage {
    description: string;
    hurdles: Hurdle[];
}

export interface Performance {
    title: string;
    text: string;
}
export interface ProjectBox {
    name: string;
    slug: string;
    heroImage: ImageField;
    thumbnail: ImageField;
    client: Client;
    titleLineOne: string;
}
export interface Project {
    name: string;
    slug: string;
    heroImage: ImageField;
    heroVideo?: string;
    titleLineOne: string;
    titleLineTwo?: string;
    client: Client;
    goLiveDate?: string;
    caseStudyTitle: string;
    caseStudyText: string;
    caseStudyImage: ImageField;
    externalUrl: string;
    rolloverDetails: string;
    homepage: boolean;
    featured: boolean;
    gallery: GalleryImage[];
    challenge: ProjectStage;
    solution: ProjectStage;
    results: ProjectStage;
    performance?: Performance[];
    testimonial?: Testimonial;
}

export interface Client {
    name: string;
    logo: ImageField;
    promoLogo: ImageField;
    featured: boolean;
    featuredProject?: string;
    id: string;
}
