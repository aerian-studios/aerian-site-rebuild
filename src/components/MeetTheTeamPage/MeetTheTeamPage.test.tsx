/// <reference types="@types/jest" />
import * as React from "react";

import * as renderer from "react-test-renderer";

import { meetTheTeam } from "../../types/fixtures";
import { MeetTheTeamPage } from "./index";

describe("MeetTheTeamPage", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(<MeetTheTeamPage page={meetTheTeam} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
