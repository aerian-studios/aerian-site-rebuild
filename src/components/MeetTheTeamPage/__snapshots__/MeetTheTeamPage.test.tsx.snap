// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeetTheTeamPage renders correctly 1`] = `
<section>
  <header
    className=""
    id="page-header"
    style={Object {}}
  >
    <div
      className=""
    >
      <figure
        className=""
      />
      <div />
    </div>
    <div />
  </header>
  <div
    className=" "
  >
    <div
      className=" "
    >
      {"name":"Paul Goodenough","jobTitle":"CEO & Founder","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/484/paolo.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/483/paolo-funny.png","description":"Paul founded Aerian to work on projects of worth, and has shaped, managed and delivered some of the biggest national and international digital projects for clients such as BBC, Channel 4, Sky, Into Film, The Science Museum, Breast Cancer Now, Dementia Friends, Alzheimer's Society, Danone, Tesco Mobile, Nickelodeon and more.","fact":"Paul is also a double Emmy nominated writer & producer for  brands inc. BBC, How to Train Your Dragon and Peppa Pig; and owns over 20k comics. Paul has also been highlighted as one of the top ten innovators in the UK by UKTI.","skills":["Accessibility and Compliance Semantics","Business Development","Conceptual Design","Consulting","Conversion Rate Optimisation","Copywriting","Flash","Graphic design","Marketing","Motion Graphics","Multivariate Testing","Partnerships","Product Development","Product Management","Prototypes","SEO","Strategy","UI Design","User Experience","User Testing","UX Design","Visual Design","Wireframes","XSLT"]}
    </div>
    <div
      className=" "
    >
      {"name":"Geoff Wells","jobTitle":"COO","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/486/geoff.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/485/geoff-funny.png","description":"Geoff has wide experience in business and marketing strategy and technology product development, with more than 25 years experience across a range of businesses and sectors. He has worked with global brands and aspiring start-ups in creating business and product strategies and bringing products to market. Clients and brands Geoff has worked with include Google, Unilever, Microsoft, IBM, Oracle, The MetOffice, BBC, CNN, Good Energy, British Chambers of Commerce, and the University of Bath.","fact":"Did you know Geoff once took a wrong turn in Seville and was mistaken for a matador. Having been escorted to the bull fight ring he said the steaks were never higher.","skills":["Agile Product Owner","Business Development","Business Operations","Consulting","CRM","Marketing","Partnerships","Product Management","Strategy"]}
    </div>
    <div
      className=" "
    >
      {"name":"Simon Neaves","jobTitle":"Technical Director  ","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/488/simon.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/487/simon-funny.png","description":"Simon has single handedly created the development team that has seen us lauded as a \\"truly first class\\" outfit and highlighted as one of the best development companies in the UK. Simon has acted as Technical Architect for some of the largest web projects in the world combining existing services with bespoke development to create extensible web projects that offer unrivalled full life-cycle value for money and maximum extensibility.\\n\\nSimon has managed and delivered hundreds of projects for our clients including BBC Mixital, CNN's Africa View App, Into Film (Festivals and Educator), Wiltshire Council's digital platform, multiple Business Growth Hubs, The Space and many more. As a highly experienced developer in his own right, but with natural tendencies for leadership and strategy, Simon stands alongside our developers as a facilitator to optimise performance and delivery for our clients.","fact":"Simon is the only person we know to have given his beard not only a first name, but also a middle name. He once not only lost his car keys but also a wheel barrow in his beard.","skills":["Agile Product Owner","Agile Project Management","Agile Scrum Master","AWS","Backbone","Consulting","Git","Hosting","Information Architecture","Javascript","Jenkins","OOP","PHP","SQL"]}
    </div>
    <div
      className=" "
    >
      {"name":"Ben Poole","jobTitle":"Director of DevOps","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/491/ben.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/489/ben-funny.png","description":"Ben has extensive experience with high availability platforms and services. Ben is one of the biggest single reasons for Aerian's massive success with projects of scale, and his ability to navigate his way through the maze of systems, people, teams and services enables us to deliver highly technical and complex projects on a regular basis.","fact":"Menial Job Peon is not only an anagram of Benjamin Poole but also Ben's previous job title.","skills":["Agile Project Management","Agile Scrum Master","AJAX","AWS","Chef","CRM","Electron","Git","Grunt","Hibernate","Hosting","HTML","Hudson","Information Architecture","Infosec","J2EE","Java","Javascript","Jenkins","jQuery","JSON","Linux/UNIX","Load Testing","Lucene","Monitoring","MySQL","Node","OOP","PHP","Product Management","Puppet","RDBMS","Server Administration  ","Shell Scripting ","SOLR","Spring","SQL","SVN","Virtualization","XML","XSLT"]}
    </div>
    <div
      className=" "
    >
      {"name":"Andy Max","jobTitle":"Senior Developer ","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/493/andy.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/492/andy-funny.png","description":"Andy is one of Aerian's development rocks with extensive skills and experience in developing and implementing CMS systems, API integrations, and CRM development. Andy has been lead developer for some of our largest and most complex projects including BBC Things To Do, Go Groups, Into Film (Festival and Educator) and Wiltshire Council.\\n\\nAndy has a BSc in Electronic Computer Engineering specialising in Internet Technology and over 10 years professional and commercial experience.","fact":"Andy's real name caused a buffer overflow which inspired the W32/Nimbda worm.","skills":["AJAX","Asterisk","C","C#","CRM","CSS","Forge","Git","Hosting","HTML","Javascript","jQuery","Linux/UNIX","Load Testing","MySQL","OOP","PAL","Perl","PHP","PostgreSQL","Server Administration  ","Shell Scripting ","SOLR","SQL","SVN","XML"]}
    </div>
    <div
      className=" "
    >
      {"name":"Anthony Butterfield","jobTitle":"Senior UI Designer","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/564/ant.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/563/ant-funny.png","description":"Anthony is a digital designer and art director with a commitment to user centred design. He has over 14 years industry experience, working within the world of Digital, User Experience, Advertising and E-learning. Designing for brands including American Airlines, British Airways, BAE Systems, Bloomberg, TfL, Barclays, Zurich, Paddy Power, Cineworld, P&O Ferries and LV insurance. Anthony is a skilled design thinker and problem solver with a passion for creating compelling, innovative and usable design experiences across multiple platforms.","fact":"Did you know Anthony once spent an hour train journey on his commute home sat oposite celebrity Hugh Bonneville (A.K.A Lord Grantham from Downton Abbey) their legs both vying for space and only noticed who it was when it was time to depart the train.","skills":["Conceptual Design","Motion Graphics","UI Design","User Experience","UX Design","Visual Design"]}
    </div>
    <div
      className=" "
    >
      {"name":"Chris Bingham","jobTitle":"Project & Admin Assistant","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/566/chris.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/565/chris-funny.png","description":"Chris provides essential quality assurance on the company's projects to ensure that they adhere to very high standards. In addition to this he contributes to Domain and SSL Management and provides help and advice to customers in this area. Chris also has a keen interest in video production; he plans and produces videos that appear on the Aerian social media channels.","fact":"Chris loves pickle so much that he named his dog Branston.","skills":["Agile Project Management","Business Operations","CSS","HTML","jQuery","Multiple Device Testing","Multivariate Testing"]}
    </div>
    <div
      className=" "
    >
      {"name":"Darren Torchia","jobTitle":"Developer ","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/498/darren.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/497/darren-funny.png","description":"Darren is a keen and passionate web software developer, adept at integrating a variety of services and technologies. He has delivered complex systems where security of personal and financial data is paramount, and systems requiring direct and time-critical communication with emergency services.","fact":null,"skills":["Git","HTML","jQuery","Linux/UNIX","MySQL","OOP","PHP","SEO"]}
    </div>
    <div
      className=" "
    >
      {"name":"David Martinez-Plaza","jobTitle":"Creative Developer ","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/502/david.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/501/david-funny.png","description":"David has years of experience creating rich and complex mobile apps, web apps, games, systems and experiences. His is a highly creative front-end developer, with skills extending across apps, web and through to PHP development.\\n\\nDavid has worked on leading web and mobile products including BBC Companion App, BBC Mixital, CNN Africa View News App, Good Energy's meter reading app, The Science Museum's Antenna project, The Space: Open Call, and The Space (Arts Council).","fact":"David is a keen cyclist, motorcycle rider and drone pilot. But not all at the same time. He's working on it.","skills":["AJAX","Android","App Development","Backbone","C++","CSS","Git","HTML","Java","Javascript","jQuery","JSON","Node","OOP","SQL"]}
    </div>
    <div
      className=" "
    >
      {"name":"Hannah Williams","jobTitle":"Client Side Developer ","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/504/hannah.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/503/hannah-funny.png","description":"Hannah has been involved in many of our most engaging and mobile responsive web projects. She is highly skilled in creating industry leading experiences across multiple devices, including the very recent launch of the BBC's footer: Discover Cards which is consumed by millions of users every day.\\n\\nHannah brings years of experience working in both client side and server side development, creating applications and experiences used and enjoyed by millions of users globally.","fact":"Hannah has a love of animals, owning a growing collection including dogs, rescue cats, chickens and soon to be who knows what (thinking horses, Alpacas). Also an up-and-coming ballet dancer and gymnast, albeit a beginner now, but maybe someday...","skills":["CSS","Electron","HTML","jQuery","JSON","PHP","SEO","SQL","SVN","XML","XSLT"]}
    </div>
    <div
      className=" "
    >
      {"name":"Gyula Soós","jobTitle":"Developer","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/735/dscf2303.jpg","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/736/dscf2317.jpg","description":"Gyula has a fresh BSc (Hons) in Applied Computing from University of Bath. While at the university he learned a little bit about everything, now he's eager to learn a lot about the key technologies. He's passionate about anything IT related: cloud technologies, computer hardware/software and digital trends.","fact":"Gyula’s colleagues have a hard time understanding him, because he implemented the Hungarian notation in spoken language. Prefixing verbs with a 'v', nouns with a 'n', adverbs with a 'd', and adjectives with a 'j' just 'vMakes jmore nsense dfor nhim'.","skills":["Agile Scrum Developer","AWS","C#","CSS","Graphic design","HTML","Javascript","Multiple Device Testing","Openstack","PHP"]}
    </div>
    <div
      className=" "
    >
      {"name":"James Pearce","jobTitle":"Client Side Developer ","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/506/james.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/505/james-funny.png","description":"Core to our front-end development team, James has worked on some of Aerian's most visually important interactive applications, apps and games.\\n\\nComing from a creative background, James' eye for interactivity and user experience ensures that the granular levels of user-experience seep through to all areas of our projects. James has delivered multiple projects including BBC Mixital, CBeebies, Sky Elementary, the Science Museum, and Arts Council which are some of our proudest achievements.","fact":"James is best known for accidentally entering a Bruce Willis lookalike competition while playing his double bagpipes outside his local public house.  ","skills":["Agile Scrum Master","App Development","Backbone","CSS","Foundation","Git","Graphic design","Grunt","HTML","Javascript","Multiple Device Testing","PHP","SVN","UI Design"]}
    </div>
    <div
      className=" "
    >
      {"name":"Jason  Goodenough","jobTitle":"Client Side Developer ","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/509/jason.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/508/jason-funny.png","description":"Jason has built the front end experiences on some of the world's biggest sites including over 100 BBC projects. His work has been widely upheld by the industry as shining examples of best practice.\\n\\nJason works as both client side developer and Quality Assurance Officer ensuring our sites are valid, performant on all browsers and devices and accessible beyond just being ‘acceptable'.","fact":"Jason is a avid follower of Swindon Town.","skills":["CSS","HTML","jQuery","XML","XSLT"]}
    </div>
    <div
      className=" "
    >
      {"name":"Jonathan Chard","jobTitle":"Technical Project Manager","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/758/dsc00096.jpg","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/757/dsc00108.jpg","description":"Jonathan's earliest memory of programming was cracking open a ZX Spectrum adventure game and fixing how the dice rolled... Ever since then he has worked with computers in some way or other and now has more than 25 years experience of working in the software industry,  working in a variety of sectors including banking, scientific software, healthcare and the pharmaceutical industry. In that time he has held a variety of roles ranging from development, business analysis, enterprise architecture and product and project management. He has also presented at various conferences in data science such as PhUSE, useR! and EARL. He has an MSc in Software Engineering from Teesside University, and a degree in Mathematics with Computation from Loughborough University.","fact":"Jon once hitched from Loughborough to Paris during Rag Week and broke the record for the number of lifts needed to complete the trip.","skills":["API design","Business Analysis","Business Development","C++","Enterprise Architecture","Git","Groovy","J2EE","Java","Javascript","Jenkins","OGS","Ontologies","OOP","Product Management","Project Management","Python","R","RDF","REST","Selenium","SGE","Spring","SVN","UML","XML","XSLT"]}
    </div>
    <div
      className=" "
    >
      {"name":"Marek Rodgers","jobTitle":"Project & Admin Assistant","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/568/marek.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/567/marek-funny.png","description":"Marek provides key project management and quality support to our clients and our team to ensure smooth delivery of projects. Marek is also responsible for QA testing, and provides input across the delivery process with the insight and understanding of an in-depth UX thinker.","fact":"Marek has more than 10 hours of recorded flight time with a Grob Tutor, sadly his high sensitivity to manoeuvres prevented him from becoming the next Top Gun.","skills":["Agile Scrum Developer","ISO 27001","ISO 9001","Multiple Device Testing","Multivariate Testing","QA & Testing"]}
    </div>
    <div
      className=" "
    >
      {"name":"Matt Kane","jobTitle":"Head of Software","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/734/dscf2391.jpg","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/752/dscf2407.jpg","description":"Matt has 20 years of web development experience, and has been creating mobile apps since the first days of iPhone and Android. He is as happy roadmapping a new product line, working on a full stack Node/Angular application or flipping bits on a microprocessor. As well as being an experienced developer, Matt is a serial entrepreneur with a flair for developing new products. He created the world's fastest camera flash, the #3 most popular Cordova plugin, and market-leading products in sectors as diverse as spaceflight photography and beekeeping.","fact":"Matt first trained as a baker and pastry chef, but retired in glory at the age of 20 because his love of coding was even greater than his love of profiteroles.","skills":[]}
    </div>
    <div
      className=" "
    >
      {"name":"Matthew Ellis","jobTitle":"Developer","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/755/dscf2338.jpg","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/754/dscf2348.jpg","description":"Matt is a recent graduate from Swansea University in Aerospace Engineering. Alongside his degree, Matt started learning to develop websites, before getting a job with a small company where he was able to enhance his understanding and develop new skills. Matt has a keen interest in cloud architecture like AWS but loves getting stuck into new and exciting projects.","fact":"When enrolling at university, Matt got confused by the IT reference to the cloud and ended up getting a degree in Aerospace Engineering.","skills":["Agile Scrum Developer","AWS","CSS","Git","HTML","Javascript","jQuery","MySQL","PHP","Python"]}
    </div>
    <div
      className=" "
    >
      {"name":"Paul Warder","jobTitle":"Senior Designer ","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/517/paul.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/516/paul-funny.png","description":"Paul has been at the forefront of our heaviest user experience design projects, delivering complex user journeys in a simple, easy to consume manner for users of all abilities.\\n\\nPaul's designs have placed Aerian at the forefront of the digital industry, helping inform best practice and creating brands and experiences for multi million pound projects including; BBC Climate Asia, BBC Whispering Stars, Tesco Mobile Challenge, David Cameron / Alzheimer's Society's Dementia Friends campaign and Into Film.","fact":"Did you know that Paul once appeared on the famous 90s show \\"Games Master\\" and competed in a Super Bomber man competition... He didn't win!","skills":["Conceptual Design","Graphic design","Motion Graphics","Multiple Device Testing","Prototypes","UI Design","User Experience","UX Design","Visual Design","Wireframes"]}
    </div>
    <div
      className=" "
    >
      {"name":"Neil Thorne","jobTitle":"Head of Design","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/521/neil.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/520/neil-funny.png","description":"Neil has worked at the highest levels in the digital design industry for over 15 years as a UI and UX designer and has extensive knowledge of front-end build practices, CRO and MVT.\\n\\nHe has powered the user-centred approach and products for a range of clients including Ricoh, Samsung, NHS, BBC Sport (and other BBC departments) and Breast Cancer Now - where he was the project lead helping to merge two sizeable charities into a single united £30m+ charity brand and website.","fact":"In 2006 Neil managed to successfully clone himself… twice. Whilst encouraged to understand the ways of Photoshop, all the clones want to do is play Minecraft.","skills":["Conceptual Design","Graphic design","Multivariate Testing","Prototypes","UI Design","User Experience","User Testing","UX Design","Wireframes"]}
    </div>
    <div
      className=" "
    >
      {"name":"Ryan Cheetham","jobTitle":"Designer","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/523/ryan.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/522/ryan-funny.png","description":"Ryan orginally started out in product design with a BA Product Design degree and moved swiftly into the digital world.\\n\\nRyan has developed keen creative and development skills, and has designed and developed digital products and platforms for leading brands including Into Film, Nikon, Ricoh, Samsung, Invisalign, Travelodge, Jury's Inn, Pearson and PixelPin.","fact":"Ryan once managed to get a selfie with a 5m great white shark! ","skills":["Conceptual Design","Conversion Rate Optimisation","CSS","Flash","Graphic design","HTML","Javascript","jQuery","Motion Graphics","Multiple Device Testing","Prototypes","SEO","UI Design","User Experience","UX Design","Visual Design","Wireframes"]}
    </div>
    <div
      className=" "
    >
      {"name":"S'unya Dickman","jobTitle":"Head of CSD","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/525/sunya.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/524/sunya-funny.png","description":"S'unya has worked on many of Aerian's most interactive and leading-edge projects including BBC Whispering Stars, BBC iWonder Guides, CNN Africa View News app, The Science Museum's Antenna and The Space: Open Call. He has a BA (Hons) Interactive Arts, and specialises in App development, HTML/CSS and JavaScript.","fact":"S'unya's grandfather invented the dictaphone (+ answering machine). He has been practicing TaiJi Quan for over half his life and this year he cycled over 6000km a year commuting.","skills":["App Development","Backbone","CSS","HTML","Javascript","React"]}
    </div>
    <div
      className=" "
    >
      {"name":"Spencer Westwood","jobTitle":"Senior Developer ","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/527/spencer.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/526/spencer-funny.png","description":"Spencer has wide technology and project management skills and experience across multiple verticals including financial services, manufacturing, airlines and government.\\n\\nSpencer is core part of our development team providing year-on-year support for BBC's Knowlege & Learning team, and has worked on BBC's WW1 at Home, BBC Food, and BBC Companion App just to name a few!!","fact":"Spencer owns six guitars, two pianos, a violin and a djembe but doesn't have a partridge in a pear tree... yet.","skills":["Agile Scrum Master","Android","App Development","Backbone","Browser Technologies","Business Development","C","C++","Consulting","Conversion Rate Optimisation","Copywriting","CRM","CSS","Electron","Forge","Foundation","Git","Glow","Grunt","Hosting","HTML","Infosec","iOS","J2EE","Java","Javascript","Jekyll","jQuery","JSON","Linux/UNIX","Lucene","Marketing","Monitoring","Motion Graphics","Multiple Device Testing","Multivariate Testing","MySQL","Node","nosql","Objective-C","PAL","Perl","PHP","PostgreSQL","Product Development","Product Management","Prototypes","Python","RDBMS","Ruby","SEO","Server Administration  ","Shell Scripting ","Siebel","SOLR","Spectrum","Spring","SQL","SVN","UI Design","User Testing","XML","XSLT"]}
    </div>
    <div
      className=" "
    >
      {"name":"Wayne Pitts","jobTitle":"Client Side Developer ","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/529/wayne.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/528/wayne-funny.png","description":"Wayne is one of the best multi-disciplinary developers in the industry, covering complex front-end build and resilient back-end application development including moderation, content management systems and version-control.\\n\\nWayne has worked his wizardry on projects such as BBC Climate Asia, Credit Union Quiz, The Film Network, Into Film (Festival and Educator), NYPP Festivals and BBC Mixital.","fact":"Wayne can't physically hold cotton wool, even thinking about this toxic material makes him squirm.","skills":["Android","Browser Technologies","CSS","Git","Grunt","HTML","Javascript","jQuery","JSON","Multiple Device Testing","MySQL","PHP","SVN","XML"]}
    </div>
    <div
      className=" "
    >
      {"name":"Aero The Dog","jobTitle":"Distractor","live":true,"imageNormal":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/530/aero.png","imageFunny":"https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/530/aero.png","description":"Bark, Bark!! Woof Woof!!!\\n\\nAero is the office dog, belonging to Hannah Williams. Leaning on years of Hannah's ability to distract, her protege, the irrepressible Aero has firmly placed himself as the chief distractor - the student has become the master!\\n\\nWith his skills in amusement and inability to walk across anything like a smooth surface, he forms an integral part of the team.","fact":"All Aero wants to do all day is roll in glorious mud whilst chasing a tennis ball and coming home to eat Lasagne. ","skills":["Fetch","High five","Lay down","Sit","Spin"]}
    </div>
  </div>
</section>
`;
